src/translations/de-ch.ts

```diff
@@ -10,10 +10,10 @@ const translation: Translation = {
   copy: 'Kopieren',
   progress: 'Fortschritt',
   resize: 'Größe ändern',
-  scroll_to_end: 'Zum Ende scrollen',
-  scroll_to_start: 'Zum Anfang scrollen',
-  select_a_color_from_the_screen: 'Wähle eine Farbe vom Bildschirm',
-  toggle_color_format: 'Farbformat umschalten'
```

Comment:
Not loving the snake case terms, eh? (It's fine, I chose them because they're a bit easier to read in templates but it's not a big deal.)

Reply:
Haha I was going with the naming convention rule. There were so few snake case pieces I figured it'd just be easier to remove. But yes, personally I am not a fan of snake case since this isn't Python. 😜 

Let me know what you would like to see.

Reply:
Totally fine. To be honest, it did feel weird since I don't use it anywhere else so we can leave it as-is.